package.json

```diff
@@ -88,5 +88,9 @@
       "eslint --ignore-pattern '!*'",
       "jest --findRelatedTests --passWithNoTests"
     ]
+  },
+  "publishConfig": {
+    "access": "public",
+    "tag": "next"
```

Comment:
Default set npm publication version to `next` unless the minor version is _Even_.